velox/external/duckdb/tpch/dbgen/build.cpp

```diff
@@ -53,10 +53,10 @@ static void gen_phone(DSS_HUGE ind, char *target, seed_t *seed) {
 	RANDOM(exchg, 100, 999, seed);
 	RANDOM(number, 1000, 9999, seed);
 
-	sprintf(target, "%02d", (int)(10 + (ind % NATIONS_MAX)));
-	sprintf(target + 3, "%03d", (int)acode);
-	sprintf(target + 7, "%03d", (int)exchg);
-	sprintf(target + 11, "%04d", (int)number);
+	snprintf(target, PHONE_LEN, "%02d", (int)(10 + (ind % NATIONS_MAX)));
```

Comment:
This duckdb code is imported into Velox, so in my view the right place to change this is in Duck db code , otherwise we will have to redo this when we reimport .

Reply:
Sure, can you advise the code pointer of the duckdb code base we should apply the change?

Reply:
Reping on my duckdb question...